_posts/2015-10-07-code-talks-2015-retrospective.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title: "code.talks 2015 retrospective"
+date: "2015-10-07 09:00:00"
+icon: comments
+categories: conferences
+authors: ["Jens"]
+---
+
+For the fifth time about 1.500 developers gathered for the yearly `code.talks` conference in Hamburg:
+on September 29 and 30 also a dozen ePagees went to the Cinemaxx Hamburg Dammtor to enjoy two days fully packed with [lots of talks](https://www.codetalks.de/2015/programm) from 13 different tracks.
```

Comment:
on September 29th and 30th, a dozen ePagees went to the Cinemaxx Hamburg Dammtor to enjoy two days fully packed with [lots of talks](https://www.codetalks.de/2015/programm) from 13 different tracks.